apps/opik-backend/Dockerfile

```diff
@@ -14,6 +14,21 @@ RUN mvn versions:set -DnewVersion=${OPIK_VERSION} && \
 ###############################
 FROM amazoncorretto:21-al2023
 
+RUN yum update -y && yum install -y shadow ca-certificates openssl perl \
+    && yum clean all
+VOLUME /tmp
+# Add bundle URL from https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html
+ADD https://truststore.pki.rds.amazonaws.com/us-east-1/us-east-1-bundle.pem /tmp/certs/global-bundle.pem
```

Comment:
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html
@liyaka can we add this certificate instead? the cert you're adding is just for `us-east-1` 
https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem